### ISSUE TITLE
Sending delete event to RabbitMQ 

### ISSUE THREAD

**Issue by adadychanka** [giveaway-mongo/categories-service#1]
Should be there a delete event emitting?

https://github.com/giveaway-mongo/categories-service/blob/master/src/modules/categories/categories.service.ts#L153

**Comment by AnvarAka1** [comment]
Yes, there should be a delete event emitter also. @Ruslan0111 will do that
